src/main/kotlin/no/nav/familie/ef/sak/behandlingshistorikk/domain/Behandlingshistorikk.kt

```diff
@@ -98,6 +98,7 @@ fun JsonWrapper?.tilJson(): Map<String, Any>? {
 }
 
 enum class StegUtfall {
+    UTREDNING_PÅBEGYNT,
```

Comment:
Er dette noe frontend må ta hensyn til?

Reply:
Frontend henter kun ut spesifikke deler av behandlingshistorikken og som backend har gjort om til hendelser som vises frem for saksbehandler. Det er kun første vilkår-innslag som sendes til frontend som "Behandling opprettet". Dette innslaget lages automatisk og det er ikke der `UTREDNING_PÅBEGYNT` dyttes inn. 

Tror ikke det skal være noen caser hvor det ikke er laget et innslag med vilkår før innslaget med `UTREDNING_PÅBEGYNT` kommer, men om noen vet så plis rop ut. Kan jo bli en sak at den første ikke skal legges inn som nå og da kan det bli problematisk